tests/tantivy_test.py

```diff
@@ -555,10 +571,10 @@ def test_create_readers(self):
 
 class TestSearcher(object):
     def test_searcher_repr(self, ram_index, ram_index_numeric_fields):
-        assert repr(ram_index.searcher()) == "Searcher(num_docs=3, num_segments=1)"
+        assert repr(ram_index.searcher()) == "Searcher(num_docs=3, num_segments=3)"
```

Comment:
Hmm, why has this changed back?

Reply:
The writer should have been created with a large enough memory limit to allow 1 segment:

```python
writer = index.writer(10_000_000, 1)
```

^^ This is the writer inside the `create_index()` that is being called from `ram_index`.  I wonder if this change is related to the tantivy upgrade itself?

Reply:
@GodTamIt Could you test whether increasing that 10M brings the `num_segments` back down to 1?

Reply:
Looks like a small behavior change. Increasing the memory limit seems to have fixed the issue.

Reply:
The memory accounting in Tantivy was improved and now tracks memory that was not included before, hence the same workload apparently needs more memory. (Actually, it needs the same, but the limit now includes more of the actual working set.)